lib/models/App.js

```diff
@@ -241,7 +241,7 @@ class App {
     let result = helper.loadRecursively(filePath, {
       required: false,
       handler: function(klass) {
-        return new klass(app)
+        return new klass(this)
```

Comment:
this should be `this. expressApp`

Reply:
this PR is to change passing this instead of this.expressApp in order to let the controller get the variable in app